lib/active_set/exporting/operation.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require_relative '../column_instruction'
+
+class ActiveSet
+  module Exporting
+    class Operation
+      def initialize(set, instructions_hash)
+        @set = set
+        @instructions_hash = instructions_hash
+      end
+
+      def execute
```

Comment:
At the moment, we only support the CSV format, but support for JSON wouldn't be difficult to add.